applications/asset_tracker_v2/src/main.c

```diff
@@ -16,7 +16,8 @@
 #endif
 
 /* Module name is used by the event manager macros in this file */
-#define MODULE app_module
```

Comment:
**question:** why is the renamed? This change does not seem to be related to using CAF.

Reply:
CAF modules initialize when main module is ready.
```
module_set_state(MODULE_STATE_READY);
```

I really have not checked it even though I should. Do you have any dependencies on this name?

Reply:
I can't tell. @jtguggedal might know.

Reply:
I don't know how the event manager ueses the `MODULE` macro internally, so not quite sure how the name change affects things in that regard. It will at least cause the registered log module to change name.

The events from this module are all prefixed `APP_` and it's referred to as "application module" in the documentation and figures, so that should probably be updated.